src/diffusers/schedulers/scheduling_dpmsolver_multistep.py

```diff
@@ -154,7 +162,9 @@ def __init__(
         algorithm_type: str = "dpmsolver++",
         solver_type: str = "midpoint",
         lower_order_final: bool = True,
+        euler_at_final: bool = False,
```

Comment:
> should we default euler_at_final to be True? you mentioned there is a tradeoff in image details but I think artifacts are much more undesirable

I would leave `euler_at_final` as `False` since the current default setting works great for SDv15. SDXL seems to be more of a special case here

> I think we should deprecate lower_order_final now we have euler_at_final- we can strongly recommend to set euler_at_final to be True when using less than 15 steps

Don't think we should deprecate it. Already for backwards compatibility reasons we'll need to keep the two and again I think we should be careful to not destroy a functioning, well-working scheduler setting for SDv15.


Essentially, what is done here is to give the user a possibility to have `lower_order_final=True` even for models where we use more than 15 inference steps, but this should not come at the expense of breaking existing workflows, so I don't think we can default `euler_at_final` to True or that we can remove `lower_order_final`


Instead of adding `euler_at_final`, we could add a parameter, called `enable_lower_order_below: int = 15` that we allow user to set to 1000 for SDXL. This might be a bit cleaner.